x/ref/runtime/internal/vtrace/vtrace_logging_test.go

```diff
@@ -113,7 +113,6 @@ func TestVIRPCWithLogging(t *testing.T) {
 	if err != nil {
 		t.Fatalf("logging call failed: %v", err)
 	}
-	expectSequence(t, *record, []string{
-		"\"\".Log: vtrace_logging_test.go:19] logging",
-	})
+	expectedSpanRegex := "\"\".Log: vtrace_logging_test.go:19] [a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}logging"
```

Comment:
you may want to add a space or colon after the uuid to make it easier to read/parse. I don't think it matters much for parsing since it's a fixed length though. Up to you.